docker/transport/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# todo: cant we use base image with ports open to trezor-user-env container?
+# todo: if no, do we want proxy like in other dockerfiles?
+
```

Comment:
Would just use a suite base ci image for this.

Reply:
That would mean running base image container inside base image container. I would like to avoid that and use 
base image that runs the actual test script with support of trezor-user-env container running inside the base image.